source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -7,6 +7,8 @@ weight: 162
 
 This is part of [going live](/switching_to_live/#go-live).
 
+Worldpay is Government Banking's payment service provider (PSP). You must be a central government or health sector organisation to use Worldpay.
```

Comment:
You must be a central gov or health sector org to use Gov Banking's contract

I think maybe it's not that Worldpay is Gov Banking's PSP but that Gov Banking provide access to a centrally procured contract with Worldpay? maybe that's minor